cluster_tools/dockered-slurm/how-to.md

```diff
@@ -0,0 +1,36 @@
+# Install segtools
```

Comment:
What is this file used for? Some parts seem outdated, and maybe it should be merged with the README?

Reply:
you are right. i removed it as the readme should contain all the relevant parts.